src/components/RootNavigator/BaseNavigator.js

```diff
@@ -139,11 +148,57 @@ export default function BaseNavigator() {
         component={AddAccountsNavigator}
         options={{ headerShown: false }}
       />
+      <Stack.Screen
+        name={NavigatorName.RequestAccount}
+        component={RequestAccountNavigator}
+        options={{
+          headerShown: false,
+        }}
+        listeners={({ route }) => ({
+          beforeRemove: () => {
+            /**
+              react-navigation workaround try to fetch params from current route params
+              or fallback to child navigator route params
+              since this listener is on top of another navigator
+            */
+            const onError =
+              route.params?.onError || route.params?.params?.onError;
+            // @TODO replace with correct error
+            if (onError && typeof onError === "function")
+              onError(
+                route.params.error ||
+                  new Error("Request account interrupted by user"),
```

Comment:
I'll prepare a live-common update :+1: not sure it will go in effect with v0 though

Reply:
We did not have planned the error handling for the Platform v0.
It will be part of the v1 specs, so, in the end, yes this should be defined somewhere in LLC, but we are not ready to do so (i was asked not to do it when i suggested it so... please don't add more features that were not planned for v0).

Reply:
https://github.com/LedgerHQ/ledger-live-common/pull/1229 once this is merged we can go back and relink the errors properly